raitools/README.md

```diff
@@ -0,0 +1,16 @@
+# Responsible AI Tools SDK for Python
+
+### This package has been tested with Python 3.6, 3.7 and 3.8
+
+The Responsible AI Tools SDK enables users to analyze their machine learning models in one API. Users will be able to analyze errors, explain the most important features, validate fairness, compute counterfactuals and run causal analysis in a single API.
+
+Highlights of the package include:
+
+- `fairness.add()` allows users to run Fairlearn to assess model fairness
```

Comment:
Nit: I'd prefer "runs Fairlearn to assess model fairness" since it's uncommon in my experience to talk about "users" in open source READMEs, but up to you

Reply:
done!